_data/devdocs/en/guides/contracts.md

```diff
@@ -210,20 +210,20 @@ all under an Apache license.
 
 {% autocrossref %}
 
-Alice is concerned about her privacy.  She knows every transaction gets
-added to the public block chain, so when Bob and Charlie pay her, they
+Alice is concerned about her privacy. She knows every transaction gets
+added to the public Blockchain, so when Bob and Charlie pay her, they
```

Comment:
Unless it's changed since I last contributed here, Bitcoin.org uses the original Nakamoto-style spelling (and capitalization) for block chain (two words, common noun).  Last time this was discussed, I think we actually settled it with a coin flip to avoid bikeshedding.